client/src/main/java/io/pravega/client/stream/ConfigMismatchException.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.client.stream;
+
+
+public class ConfigMismatchException extends IllegalArgumentException {
+    private static final long serialVersionUID = 1L;
+    public ConfigMismatchException(String readerGroupScopedName, ReaderGroupConfig currentConfig) {
```

Comment:
Nit: missing blank line before constructor.

Reply:
fixed it